testing/local-amp-chrome-extension/popup.js

```diff
@@ -27,4 +27,14 @@ document.addEventListener('DOMContentLoaded', function() {
   if (!background.disabled) {
     switchButton.checked = true;
   }
+
+  const baseUrlInput = document.getElementById('base-url');
+  baseUrlInput.placeholder = background.defaultBaseUrl;
+  baseUrlInput.value = background.baseUrl;
+  baseUrlInput.addEventListener('input', () => {
+    background.baseUrl = document.getElementById('base-url').value;
+    if ( '/' !== background.baseUrl.substr( -1 ) ) {
```

Comment:
`background.baseUrl = background.baseUrl.endsWith('/') ? '' : '/'`
and to be safe we should actually be parsing the hostname and port from the URL in case the user enters something like `http://localhost:8000/hello?param=true` in which case the current code would add a `/` to it, but that's probably out of scope for this

Reply:
Great point. Fixed in 7bb35d4.